src/overlays/actors/ovl_Bg_Ctower_Rot/z_bg_ctower_rot.h

```diff
@@ -9,9 +9,9 @@ typedef void (*BgCtowerRotActionFunc)(struct BgCtowerRot*, GlobalContext*);
 
 typedef enum {
     /* 0 */ CORRIDOR,
-    /* 1 */ MAIN_DOOR, // Main door controls cutscene flow (updating Link fields and playing sounds). It is the one on
-                       // the left when entering from the Lost Woods
-    /* 2 */ DOOR,
+    /* 1 */ STONE_DOOR_MAIN, // Main door controls cutscene flow (updating Link fields and playing sounds). 
+                            // It is the one on the left when entering from the Lost Woods
+    /* 2 */ STONE_DOOR,
```

Comment:
What about prefixing those with something like `BGCTOWER_` or similar?

Reply:
Especially since they're in the header.

Reply:
Ah, I did it for `Gear`, but must have forgot for `Rot`. Prefixed with `BGCTOWERROT_`.